src/main/resources/blockly/js/mcreator_blockly.js

```diff
@@ -90,6 +90,28 @@ function jsonToBlocklyDropDownArray(json) {
     return retval;
 }
 
+// Helper function to use in Blockly extensions that register one data list selector field to update contents of another
+// The block may define input called "<targetName>Field" to customize field's position
+// Note that the source field must be inserted before the target field for their values to be loaded properly
+function appendAutoReloadingDataListField(sourceName, targetName, targetList) {

```

Comment:
Since this method is currently only used by your extension, it can be present next to the extensions you defined, similar how e.g. "validateResourceLocationFields" is defined in that file

Reply:
And then moved back in this file if considered more useful?

Reply:
Yes, if we need ti later somewhere else

Reply:
:heavy_check_mark: Fixed